pkg/trace/api/api.go

```diff
@@ -482,7 +489,10 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 		log.Errorf("Cannot decode %s traces payload: %v", v, err)
 		return
 	}
-	r.replyOK(v, w)
+	if n, ok := r.replyOK(v, w); ok {
+		tags := append(ts.AsTags(), "endpoint:traces_"+string(v))
+		metrics.Histogram("datadog.trace_agent.receiver.rate_response_bytes", float64(n), tags, 1)
```

Comment:
Arguably `serve_traces_bytes` would also work well here.

Reply:
Yeah, having "rate" in the name of the metric is a bit confusing. How about `vx_traces_response_bytes`?

Reply:
Changed to `serve_traces_bytes`.